addons/l10n_in/views/res_config_settings_views.xml

```diff
@@ -29,7 +29,8 @@
                         </setting>
                         <setting name="l10n_in_tds_tcs"
                                  help="Enable this to activate Tax Deduction Source and Tax Collection Source."
-                                 string="TDS and TCS">
+                                 string="TDS and TCS"
+                                 documentation="/applications/finance/fiscal_localizations/india.html">
```

Comment:
is there a doc branch to explain TDS?  @shid-odoo

Reply:
@jco-odoo Not yet, we will do it with this task -https://www.odoo.com/odoo/project/967/tasks/4143155